tests/unit/test_secure_email.py

```
@@ -0,0 +1,650 @@
+            'smtp_port': 587,
+            'username': 'user@example.com',
+            'password': 'wrong_password',
+            'use_tls': True,
+            'use_ssl': False,
+            'timeout': 30
+        }
+        
+        with patch('smtplib.SMTP') as mock_smtp:
+            mock_smtp_instance = Mock()
+            mock_smtp.return_value = mock_smtp_instance
+            mock_smtp_instance.login.side_effect = smtplib.SMTPAuthenticationError(535, "Authentication failed")
```

Ruff (0.11.9)

Undefined name `smtplib`

(F821)